Documentation/ContentObjects/Content/Index.rst

```diff
@@ -311,19 +202,130 @@ stdWrap
 .. _cobj-content-cache:
 
 cache
-=====
+-----
 
 .. ### BEGIN~OF~TABLE ###
 
 .. include:: ../../DataTypes/Properties/Cache.rst.txt
 
 .. ###### END~OF~TABLE ######
 
+Examples
+========
+
+CONTENT explained in detail
+----------------------------
+
+See PHP class :php:`\TYPO3\CMS\Frontend\ContentObject\ContentContentObject`
+for details on code level.
+
+Preamble:
+
+.. code-block:: typoscript
```

Comment:
This looks strange as a code block, maybe an admonition is a better way to emphasize the importance of that note?

Reply:
I think we don't need that anymore. CONTENT is not really the place to explain what TypoScript is in general. This is probably very old and has been copied ever since

Reply:
removed it